src/containers/PreviewDraftPage/LanguageSelector.tsx

```diff
@@ -6,36 +6,67 @@
  *
  */
 
+import { useMemo } from "react";
 import { useTranslation } from "react-i18next";
 import { useNavigate, useParams } from "react-router-dom";
-import styled from "@emotion/styled";
-import { toPreviewDraft } from "../../util/routeHelpers";
-
-const StyledSelect = styled.select`
-  margin: 0 auto;
-  width: 100%;
-  display: block;
-`;
+import { createListCollection } from "@ark-ui/react";
+import { SelectContent, SelectLabel, SelectRoot, SelectValueText } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
+import { GenericSelectItem, GenericSelectTrigger } from "../../components/abstractions/Select";
+import { routes } from "../../util/routeHelpers";
 
 interface Props {
   supportedLanguages: string[];
 }
 
+const StyledGenericSelectTrigger = styled(GenericSelectTrigger, {
+  base: {
+    width: "100%",
+  },
+});
+
+const StyledSelectContent = styled(SelectContent, {
+  base: {
+    color: "text.default",
+  },
+});
```

Comment:
Mistenker at dette ikke trengs?

Reply:
Hvis man fjerner det arver select fargen fra Hero, altså hvit. Da blir det på hvitt på hvitt! Må isåfall endres i frontend-packages.